data/systemd/Makefile

```diff
@@ -33,6 +37,7 @@ install: $(SYSTEMD_UNITS)
 	install -m 0644 -t $(DESTDIR)/$(SYSTEMDSYSTEMUNITDIR) $^
 	install -d -m 0755 $(DESTDIR)/$(LIBEXECDIR)/snapd
 	install -m 0755 -t $(DESTDIR)/$(LIBEXECDIR)/snapd snapd.core-fixup.sh
+	install -m 0755 -t $(DESTDIR)/$(LIBEXECDIR)/snapd snapd.manage
```

Comment:
Hmm, we have a management script that is named similarly already. Is this too close to be safe?

Reply:
Hm, good point. We have `snap-mgmnt`. /me scratches head.

Reply:
Just look at my suggestion below :-)